From ac39bcebfc977f095e72d14b91294e3562400d2e Mon Sep 17 00:00:00 2001
From: knuxify <knuxify@gmail.com>
Date: Tue, 6 Jun 2023 15:01:58 +0200
Subject: [PATCH] sidebar hotfix

---
 src/sidebar.py | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/sidebar.py b/src/sidebar.py
index f58b74e..1976ec9 100644
--- a/src/sidebar.py
+++ b/src/sidebar.py
@@ -243,6 +243,10 @@ class EartagFileList(Gtk.ListView):
         self._ignore_unselect = True
 
         if not self.selection_mode:
+            if not self.file_manager.selected_files:
+                self.selection_model.unselect_all()
+                self._ignore_unselect = False
+                return
             new_index = find_in_model(self.selection_model,
                 self.file_manager.selected_files[0])
             if new_index is None:
-- 
2.41.0

